### ISSUE TITLE
Windows Terminal Preview maybe others help ... help

### ISSUE THREAD

**Issue by AddictArts** [mgunyho/tere#81]
In Terminal Preview the `?` does not trigger the `Help` action, instead it searches.

To work around this use ` --map Ctrl-h:Help` or any other mapping, I believe a modifier like `Ctrl` is needed others it will search.

Or

```
   5   │ function Invoke-Tere() {
   6   │     $result = . (Get-Command -CommandType Application tere) --map Ctrl-h:Help $args
   7   │
   8   │     if ($result) {
   9   │         Set-Location $result
  10   │     }
  11   │ }
  12   │ Set-Alias tere Invoke-Tere
```

**Comment by mgunyho** [comment]
Hmm, interesting. Does it happen with a non-preview version of the terminal?

If you are able to compile Rust code yourself, could you try to clone [this repo](https://github.com/canop/crokey), and run `cargo run` in the folder `examples/print_key` and see what pressing `?` prints?

**Comment by holly-hacker** [comment]
I noticed the same thing in Alacritty on Windows. Using the crokey example, I get the following output when pressing <kbd>?</kbd> (using <kbd>SHIFT</kbd> and <kbd>/</kbd>), <kbd>ESC</kbd> and <kbd>CTRL+C</kbd> in order.
```
C:\Users\-snip-\Projects\Other\crokey\examples\print_key〉cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `C:\Users\-snip-\Projects\Other\crokey\target\debug\print_key.exe`

Type any key combination (remember that your terminal intercepts many ones)
There's no help on this app
You typed Esc
Arg! You savagely killed me with a Ctrl-c
```

**Comment by mgunyho** [comment]
So it seems that the question-mark gets correctly passed to the terminal. The crokey example handles both `key('?')` and `key(shift-'?')`. Perhaps the Windows terminal sends the latter? If you run `tere` with `--map "shift-?:Help"`, does it work?

**Comment by holly-hacker** [comment]
Passing the --map parameter seems to fix the problem on both Alacritty and Windows Terminal.

**Comment by mgunyho** [comment]
This is now fixed in v1.3.1.
